eng/Version.Details.xml

```diff
@@ -63,5 +63,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>044fa99d22e576fcfaf63f09ee9bba821855dddb</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview.19073.11">
+      <Uri>https://github.com/dotnet/corefx</Uri>
+      <Sha>0000</Sha>
```

Comment:
Don't we need the correct SHA for the chosen version?

Reply:
It will be updated by the bot on the next run. Commit sha isn't actually used by anything (yet).